docs/docs/user-guides/use-cases/nat-gateway/_whitelist-sso.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Restrict SSO to Firezone Users
+sidebar_position: 1
+description: Configure your identity provider to whitelist traffic from Firezone's WireGuard® based secure access platform.
+---
+
+# Restrict application authentication to VPN users
+
+Some identity providers support restricting SSO functionality
+Some identity providers support restricting SSO access to a predefined CIDR range or list of IP addresses. You can set this to the IP address of
+your Firezone gateway to restrict SSO to users connected to Firezone.
```

Comment:
Did this draft end up in this PR somehow?

Reply:
https://github.com/firezone/firezone/pull/1185 was merged into this PR. Was trying to consolidate all the docs changes into 1 PR.